open-company/open-company-web
# Sentry upgrade and catch wrapper

NB: wait for this cljsjs PR to be merged in https://github.com/cljsjs/packages/pull/1941

Code to add:
`(throw (js/Error. "An error occurred..."))`

To test:
- make sure you have a Sentry DSN configured for the web project where you plan to test this
- in a component you are sure that is being mounted add the above code line in the `did-mount` handler
- load the app
- :boom: you should see an error (or more) in the console with the above message
- check on sentry.io that the error is reported
- merge
